WMF Framework/PageNamespace.swift

```diff
@@ -0,0 +1,32 @@
+// https://en.wikipedia.org/wiki/Wikipedia:Namespace
+public enum PageNamespace: Int {
```

Comment:
I also noticed there's a MWKTitleNamespace enum in the project, doesn't look like it's used though (and in obj-c), maybe we can clean it out?